readme.md

```diff
@@ -84,6 +84,12 @@ Feel free to write to our [Support Center](http://devexpress.com/support/center)
   - [DxContextMenu](#dxcontextmenu)
     - [Add a Scroll Bar](#add-a-scroll-bar)
   - [DxMenu](#dxmenu)
+    - [Remove Item Paddings](#remove-item-paddings)
+    - [Place an Icon Above Item Text](#place-an-icon-above-item-text)
+    - [Specify Item Width](#specify-item-width)
+    - [Change Item Font Size](#change-item-font-size)
+    - [Wrap and Center Item Text](#wrap-and-center-item-text)
+    - [Change Item Text Color](#change-item-text-color)
   - [DxTreeView](#dxtreeview)
```

Comment:
Please apply the following change below:

Line 219:
`#### Change the Header Cells' Background Color`

Please change "Cells' Background Color" to "Cells Background Color".